README.md

```diff
@@ -2,3 +2,6 @@
 Learning about Github
 
 Hello my name is Joelle and I am learning about how Github works!
+
+
+Hi! These are new edits! yay!!!
```

Comment:
too many exclamation points for the professional environment

Reply:
Great point boss, I will update that. Sorry for my lack of professionalism